post.html

```diff
@@ -0,0 +1,253 @@
+                            Barcelona</p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304" height="176" src="photos\2.jpg" alt="Hello world">
+                    <div>
+                        <p class="text_after_images">Hello world, or, in other words, why this blog exists
+                        </p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304" height="176" src="photos\3.jpg"
+                        alt="regarding how wework">
+                    <div>
+                        <p class="text_after_images">Here are some things you should know regarding how we work </p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304" height="176" src="photos\4.jpg" alt="remote work">
+                    <div>
+                        <p class="text_after_images">Updating list of 50+ sources on distributed teams, remote
+                            work, and
+                            how to make it all work better</p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304" height="176" src="photos\5.jpg"
+                        alt="A few words about this blog platform">
+                    <div>
+                        <p class="text_after_images">A few words about this blog platform, Ghost, and how this site was
+                            made</p>
+                    </div>
+                </div>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304" height="176" src="photos\6.jpg"
+                        alt="modern remote working">
+                    <div>
+                        <p class="text_after_images">How modern remote working tools get along with Old School
+                            Cowboy's methods
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="sign_up">
+            <h2 class="sign_up_caption">Sign up for the newsletter</h2>
+            <p class="sign_up_text">If you want relevant updates occasionally, sign up for the private newsletter. Your
+                email is never shared.</p>
+            <form class="mail">
+                <input class="input_mail" placeholder="Enter your email..." type="text"><button class="mail_text">sign
+                    up</button>
+            </form>
+        </section>
+        <footer class="footer_black_square">
+            <div class="square_first_text">
+                <div class="bold_text scroll_text">DIGITAL PRODUCT DESIGN</div>
+                <div class="scroll_text" class="work">REMOTE WORK</div>
```

Comment:
double class attribute. fix here and everywhere else
```suggestion
                <div class="scroll_text work">REMOTE WORK</div>
```